src/main/java/emu/grasscutter/game/quest/exec/ExecUnlockPlayerWorldScene.java

```diff
@@ -0,0 +1,23 @@
+package emu.grasscutter.game.quest.exec;
+
+import emu.grasscutter.data.common.quest.SubQuestData.QuestExecParam;
+import emu.grasscutter.game.quest.GameQuest;
+import emu.grasscutter.game.quest.QuestValueExec;
+import emu.grasscutter.game.quest.enums.QuestExec;
+import emu.grasscutter.game.quest.handlers.QuestExecHandler;
+import emu.grasscutter.server.packet.send.PacketPlayerWorldSceneInfoListNotify;
+
+@QuestValueExec(QuestExec.QUEST_EXEC_UNLOCK_PLAYER_WORLD_SCENE)
+public class ExecUnlockPlayerWorldScene extends QuestExecHandler {
+    @Override
+    public boolean execute(GameQuest quest, QuestExecParam condition, String... paramStr) {
+        int sceneId = Integer.parseInt(paramStr[0]);
+
+        //set the scene to locked: false
+        quest.getOwner().getUnlockedScenes().put(sceneId, false);
```

Comment:
Might be good to move the logic for actually locking/unlocking the scene to the playerProgressManager, since for example activity actions can also lock/unlock world scenes.
Also when something like owner is used multiple times, it might be good to put it in a variable and use that, instead of the getter multiple times.